src/moai-core/MOAITestMgr.cpp

```diff
@@ -69,6 +70,7 @@ int MOAITestMgr::_assert ( lua_State* L ) {
 	}
 	return 0;
 }
+#endif
```

Comment:
instead of ifdefing this out, couldn't we just rename _assert to _moaiassert or something. This will mean testmgr will still function on android